packages/builtin_test/_test.pony

```diff
@@ -7,6 +7,7 @@ so `builtin` doesn't have to depend on the `PonyTest` package.
 
 use "ponytest"
 use "collections"
+use "debug"
```

Comment:
This should be removed, yes?

Reply:
this shouldn't have been there in the first place.